test/ContextManagerIntegrationTest.py

```diff
@@ -22,17 +22,21 @@ def testBasicContextualFollowup(self):
             .require("Location")\
             .build()
 
-        self.engine.register_intent_parser(intent1)
-        self.engine.register_intent_parser(intent2)
-
         self.engine.register_entity("what time is it", "TimeQuery")
         self.engine.register_entity("seattle", "Location")
         self.engine.register_entity("miami", "Location")
 
         self.engine.register_entity("weather", "WeatherKeyword")
 
+        self.engine.register_intent_parser(intent1)
```

Comment:
I think we've talked about this. This is breaking behavior. This test should still function as written. Please do not change it.